data/actionsmap/yunohost.yml

```diff
@@ -436,7 +436,7 @@ app:
                     help: Custom name for the app
                 -a:
                     full: --args
-                    help: Serialize arguments for app installation
+                    help: Serialized arguments for app script (i.e. "domain=domain.tld&url_path=/path")
```

Comment:
I prefer use [`url_path`](https://github.com/M5oul/sonerezh_ynh/blob/master/scripts/install#L6) and [`src_path`](https://github.com/M5oul/sonerezh_ynh/blob/master/scripts/install#L34) variables to be more specific and avoid confusions between `path` and `final_path` variables in the install script.